x/outline-connectivity-app/go.mod

```diff
@@ -0,0 +1,40 @@
+module outline_sdk_connectivity_test
```

Comment:
This should match the directory name

Reply:
`github.com/Jigsaw-Code/outline-sdk/x/outline-connectivity-app`

Reply:
has this been updated?